### PR TITLE
Enabled portfw'ing for iptables

### PR DESCRIPTION
**Pull request by baude**
when using the iptables firewall driver, we can no forward ports from
the container host to the container.

i.e. curl http://localhost:8080

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: src/commands/setup.rs [hunk 1]
```diff
@@ -80,13 +81,57 @@ impl Setup {
...
+                        Some(i) => {
+                            let localnet_path = format!("net.ipv4.conf.{}.route_localnet", i);
+                            let sysctl_localnet = get_sysctl_value(localnet_path.as_str())?;
+                            if sysctl_localnet != *"1" {
+                                set_sysctl_value(localnet_path.as_str(), "1")?;
+                            }
+                        }
+                    }
+                    firewall_driver.setup_network(network.clone(), id_network_hash.clone())?;
+                    let port_bindings = network_options.port_mappings.clone();
```
Comment (flouthoc) [inline src/commands/setup.rs:108]: nit: Not a blocker: Would it be possible to move this inside `firewall` create as a new function and call and single function in setup.

**Review by flouthoc — COMMENTED** [review]
@baude Could you share a tryout example as well. It would be helpful to test this.

File: src/commands/setup.rs [hunk 2]
```diff
@@ -62,7 +63,7 @@ impl Setup {
         let mut response: HashMap<String, types::StatusBlock> = HashMap::new();
 
         // Perform per-network setup
-        for (net_name, network) in network_options.network_info {
+        for (net_name, network) in network_options.network_info.iter() {
```
Comment (flouthoc) [inline src/commands/setup.rs:66]: Is `iter` needed here since i cant the change in `network_info`

File: src/commands/setup.rs [hunk 3]
```diff
@@ -80,13 +81,57 @@ impl Setup {
                     //Configure Bridge and veth_pairs
                     let status_block = network::core::Core::bridge_per_podman_network(
                         per_network_opts,
-                        &network,
+                        network,
```
Comment (flouthoc) [inline src/commands/setup.rs:84]: Same there is a type change i guess because of iter.

File: src/commands/setup.rs (hunk 2 above)
Comment (baude) [reply src/commands/setup.rs:66]: yes, it is needed

File: src/commands/setup.rs (hunk 3 above)
Comment (baude) [reply src/commands/setup.rs:84]: yes

File: src/commands/setup.rs (hunk 1 above)
Comment (baude) [reply src/commands/setup.rs:108]: i think this is a good idea, just want to confirm with @mheon

**Comment by flouthoc** [comment]
LGTM. @Luap99 @mheon PTAL . I haven't tried running yet but code looks good.

**Pull request merged**